js/interface.js

```diff
@@ -0,0 +1,64 @@
+var appDataSources = [];
+Fliplet.DataSources.get({ appId: Fliplet.Env.get('masterAppId'), attributes: ['id', 'name'] }).then(function (DSs) {
+  appDataSources = DSs.map(el => {
+    return { value: el.id, label: el.name };
+  });
+  Fliplet.Widget.generateInterface({
+    title: 'File list',
+    fields: [
+      {
+        name: 'dataSource',
+        type: 'dropdown',
+        label: 'Data Source',
+        options: appDataSources,
+        default: '',
+        change: function (value) {
+          Fliplet.Helper.field('columnName').toggle(value);
+          if (value) {
+            Fliplet.DataSources.getById(value, {
+              attributes: ['columns']
+            }).then(function (columns) {
+              $(document).find('#columnName').html('')
```

Comment:
You can just use the selector directly without the `$(document)` part

```suggestion
              $('#columnName').html('')
```

Reply:
Done